README.md

```diff
@@ -1,4 +1,33 @@
-# Azure API Manager Blue Green Deploy Function App
+# Azure Search Index Refresh Function App
+
+## Overview
+
+The purpose of this function app is to allow Azure Search indexes to be recreated without any down time. This is achieved by having a fixed endpoint for the search index managed as an API in Azure API Manager. Behind this there will exist an index in Azure Search which conforms to a naming convention captured by the following regular expression:
+
+``` RegEx
+(?<Name>.*)-(?<Version>[0-9]+-[0-9]+|)-(?<Deployment>a|b)-(?<Environment>dev|int|stag|prod)
+```
+
+### Example
+
+API Manager API
+
+* name: service-search-organisations
+* external URL: https://api.nhs.uk/service-search/search
+
+Before the function app has run:
+
+* serviceUrl: https://nhsuksearchprodne.search.windows.net/indexes/organisationlookup-a-dev/docs/
+
+after the function app has run:
+
+* serviceUrl: https://nhsuksearchprodne.search.windows.net/indexes/organisationlookup-b-dev/docs/
+
+The index has switched from _a_ to _b_ and _b_ will have been (re)created and indexed from the data source.
+
+You might want to use this approach when Azure Search synchronisation between source data and the search index can not be relied upon. Our particular scenario involved the identifiers for a given organisations not being consistent across data uploads into the data source which, without this approach, resulted in duplicate entries in the index.
```

Comment:
Suggestion to change the last sentence to something like `For example, should the underlying data not have unique identifiers, resulting in duplicate entries within the index.`